lists/src/main/java/org/odk/collect/lists/RecyclerViewUtils.kt

```diff
@@ -14,4 +16,11 @@ object RecyclerViewUtils {
 
         return itemDecoration
     }
+
+    fun RecyclerView.ViewHolder.setItemViewLayoutParams() {
```

Comment:
The goal here is to expand the view horizontally right? If so it would be good to have a name that reflects the intention, for example: `expandItemViewHorizontally` or `stretchItemViewHorizontally`